evcorona/villareal-express-api
# Feature/testing

## PR Description

Testing Framework Installation

- Add jest and supertest dependencies
- Add an API test file.
   - Check if the server runs successfully
- Add run test command to GitHub Actions

### Screen Shots/Recordings

![image](https://user-images.githubusercontent.com/73207023/231896010-91e9e42b-dde0-4863-b7ea-d7584dd04c2c.png)
